### PR TITLE
make illuminate obsolete

### PR DESCRIPTION
**Pull request by key2peace**

### CODE REVIEW

File: src/Discord/functions.php [hunk 1]
```diff
@@ -110,3 +110,41 @@ function getColor($color = 0): int
 
 	return 0;
 }
+
+/**
+ * Replacement for Illuminate\Support\Str::contains.
```
Comment (davidcole1340) [inline src/Discord/functions.php:115]: `Checks if a string contains an array of phrases`?

File: src/Discord/functions.php [hunk 2]
```diff
@@ -110,3 +110,41 @@ function getColor($color = 0): int
...
+ *
+ * @param string $string The string to check.
+ * @param array $matches Array containing one or more phrases to match.
+ *
+ * @return bool
+ */
+function contains(string $string, array $matches): bool
+{
+    foreeach($matches as $match) {
```
Comment (davidcole1340) [inline src/Discord/functions.php:124]: `foreeach`

File: src/Discord/functions.php [hunk 3]
```diff
@@ -110,3 +110,41 @@ function getColor($color = 0): int
...
+        if (strpos($string, $match) !== false) {
+            return true;
+        }
+    }
+
+    return false;
+}
+
+/**
+ * Replacement for Illuminate\Support\Str::studly.
```
Comment (davidcole1340) [inline src/Discord/functions.php:134]: `Converts a string to studlyCase`?

**Review by davidcole1340 — CHANGES_REQUESTED** [review]

File: src/Discord/functions.php (hunk 2 above)
Comment (key2peace) [reply src/Discord/functions.php]: lol yeah, forgot that, had it fixed in my running tree, not just yet here...

**Comment by davidcole1340** [comment]
Cheers

**Pull request merged**